examples/css2d_label.html

```diff
@@ -102,6 +102,7 @@
 				labelRenderer.setSize( window.innerWidth, window.innerHeight );
 				labelRenderer.domElement.style.position = 'absolute';
 				labelRenderer.domElement.style.top = 0;
+				labelRenderer.domElement.style.left = 0;
```

Comment:
I don't think defining `left` is necessary. What is your motivation for adding it?

Reply:
> I don't think defining `left` is necessary. What is your motivation for adding it?

The `left` style for lower version browser. Because lower version android system's webview doesn't have default `left` value.

Reply:
> Because lower version android system's webview doesn't have default left value.

Okay, I see. Can you please specify the highest Android version that still requires this change?

Reply:
> > Because lower version android system's webview doesn't have default left value.
> 
> Okay, I see. Can you please specify the highest Android version that still requires this change?

TBH, our testing mobile phone uses Chrome 44 and the system is out-of-date.
I tested without defining `top` in desktop Chrome 81, the demo can't work.
I saw the `webgl_loader_pdb.html` has defining `left`, just defining `left` for `css2d_label.html`.

Reply:
To sum up: I still see no reason for adding the `left` property to the code.

Reply:
> Are you sure? It seems webgl_loader_pdb only defines the top property.
Sorry, I saw the files changed, it doesn't exist.